src/inspector/node_protocol.pdl

```diff
@@ -37,3 +37,59 @@ experimental domain NodeTracing
   # Signals that tracing is stopped and there is no trace buffers pending flush, all data were
   # delivered via dataCollected events.
   event tracingComplete
+
+# Supports additional targets discovery and allows to attach to them.
+experimental domain NodeTarget
+
+  type TargetID extends string
+
+  # Unique identifier of attached debugging session.
+  type SessionID extends string
+
+  type TargetInfo extends object
+    properties
+      TargetID targetId
+      string type
+      string title
+      string url
+
+  # Sends protocol message over session with given id.
+  command sendMessageToTarget
+    parameters
+      string message
+      # Identifier of the session.
+      SessionID sessionId
+
+  # Controls whether to automatically attach to new targets which are considered to be related to
+  # this one. When turned on, attaches to all existing related targets as well. When turned off,
+  # automatically detaches from all currently attached targets.
+  command setAutoAttach
+    parameters
+      # Whether to auto-attach to related targets.
+      boolean autoAttach
+      # Whether to pause new targets when attaching to them. Use `Runtime.runIfWaitingForDebugger`
+      # to run paused targets.
+      boolean waitForDebuggerOnStart
```

Comment:
As I see it, we have two ways of proceeding with the worker inspection:

1) Consider NodeTarget to be a duck-typed Target domain

This implies that flattened operation is in place. Flatten should not be hard for you to add, simply copy protocol from here: https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/inspector/browser_protocol.pdl?q=browser_protocol&sq=package:chromium&g=0&l=5950 and add corresponding sniffing logic. When we move the front-end to be flatten by default, it will 'just work'.

2) Name things with what they are and rename this domain to NodeWorker.

Basically rename all the 'target' mentions here with the corresponding 'worker' mentions. Then you don't need to add flattened operation, Target domain is special and it adds the sessionId aspect into the protocol, other domains, including NodeWorker would not be special, so they would still need to wrap the messages. That's fine. We would still be lying to ourselves because we would be creating a target on the front-end for each worker, but I think that is tolerable.

@dgozman: wdyt?